core/open.py

```diff
@@ -95,15 +94,20 @@ def run(self):
                 )
             )
 
-        result = subprocess.call(
-            " ".join([self.open_command, file_to_open]), shell=True
-        )
-        if result == 0:
-            print("Opened " + file_to_open)
+        if self.args.print_model:
+            with open(file_to_open, "r") as f:
+                print(f.read())
```

Comment:
@jgillies Any ideas for a way to test this in our integration test suite?

Reply:
@mikekaminsky Unfortunately, no. Python isn't really my bag, so I was kind of winging it here.

Reply:
Haha okay i'll take a look either this afternoon or tomorrow and see if I can come up with something :)